installer/vagrant/debian.sh

```diff
@@ -1,5 +1,68 @@
 #!/bin/bash
 
+showhelp() {
+    echo "Usage: ubuntu.sh [options]
+    -h, --help, -?
+                Display usage information
+    --hostname=HOSTNAME
+                Appends the hostname to the guest's /etc/hosts
+                "
+    exit 0
+}
+
+# On some boxes, the hostname is not set correctly, causing RabbitMQ to fail to
+# start. This adds the hostname to /etc/hosts, which allows it to work
+# correctly.
+hostname=
+while :; do
+    case "$1" in
+        -h|--help|\?)
+            showhelp
+            ;;
+        --hostname|-n)
+            if [ "$2" ]; then
+                hostname=$2
+                shift 2
+                continue
+            else
+                echo 'ERROR: Must specify a non-empty "--hostname HOSTNAME" argument.' >&2
+                exit 1
+            fi
+            ;;
+        --hostname=?*)
+            hostname=${1#*=} # Delete everything up to = and assign the remainder
+            ;;
+        --hostname=)
+            echo 'ERROR: Must specify a non-empty "--hostname HOSTNAME" argument.' >&2
+            exit 1
+            ;;
+        *)
+            break
+    esac
+    shift
+done
+
+exists=$(grep "$hostname" /etc/hosts)
+if [ -n "$hostname" -a -z "$exists" ]; then
+    echo "Appending $hostname to /etc/hosts"
+    echo "$hostname" >> /etc/hosts
+fi
+
+
 DEBIAN_FRONTEND=noninteractive apt-get -y -m --force-yes install alsa-utils
 usermod -a -G audio vagrant
 usermod -a -G audio www-data
+
+# Fix wrong default locale
+update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8
```

Comment:
It looks like I can get rid of the locale hack in #670 if we do it here :)

Reply:
We only do it for Vagrant systems, so physical systems set up will not have any locale set

Reply:
Maybe we should do it in the installer ? It would also fix #317

Reply:
I would be against it, purely because it then forces the use of en_US.UTF-8. For example I use en_ZA.UTF-8 because it helps set things like decimal, etc which would then not be possible in the container running LibreTime. Perhaps we should detect unsupported locale and fail the install instead?

Reply:
Another option would be to attempt to detect either an unset or unsupported (ie non-UTF8 locale) and then ask them if they want to set it to en_US.UTF-8 or inform them they need to manually set a UTF-8 locale and provide instructions.

Reply:
Also I tested ubuntu-xenial with the PYTHONIOENCODING set and playout is still failing with the same error even though the default locale is ('en_US', 'UTF-8').

Reply:
Ok so I rebooted the VM and it started airtime-playout so evidently the fix wasn't applied until after the system was brought back up which I guess could make sense since it is an environmental variable being set but I'm not sure what the fix is to make it work out of the box (which is desirable for a dev VM)

Reply:
Yeah I started with a clean install, and it happens, if I halt and bring the vm back up it works but it still fails upon first boot so we need to figure out if there is a way to implement the override earlier in the boot process or an alternative way of getting it to work.

Reply:
That is very weird... A clean install for me on Ubuntu Xenial works fine... What is the output of `systemctl status airtime-playout` when it fails (I don't need the log, just the top details section)

Reply:
It's the exact same issue
```
● airtime-playout.service - Airtime Playout Service
   Loaded: loaded (/etc/systemd/system/airtime-playout.service; enabled; vendor 
   Active: failed (Result: start-limit-hit) since Wed 2019-01-16 04:46:04 PST; 9
  Process: 31118 ExecStart=/usr/bin/airtime-playout (code=exited, status=1/FAILU
 Main PID: 31118 (code=exited, status=1/FAILURE)

Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Unit
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Fail
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Serv
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: Stopped Airtime Playout Servi
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Star
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: Failed to start Airtime Playo
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Unit
Jan 16 04:46:04 ubuntu1604.localdomain systemd[1]: airtime-playout.service: Fail
```

You can see where the locale setting gets reset upon bringing the VM back up
```
2019-01-16 04:46:02,475 [__main__] [INFO ]  New locale set to: en_US.ISO8859-1
2019-01-16 04:46:02,475 [__main__] [ERROR]  Need a UTF-8 locale. Currently 'iso8859-1'. Exiting...
2019-01-16 04:46:03,137 [__main__] [INFO ]  New locale set to: en_US.ISO8859-1
2019-01-16 04:46:03,137 [__main__] [ERROR]  Need a UTF-8 locale. Currently 'iso8859-1'. Exiting...
2019-01-16 04:46:03,598 [__main__] [INFO ]  New locale set to: en_US.ISO8859-1
2019-01-16 04:46:03,598 [__main__] [ERROR]  Need a UTF-8 locale. Currently 'iso8859-1'. Exiting...
2019-01-16 04:46:04,105 [__main__] [INFO ]  New locale set to: en_US.ISO8859-1
2019-01-16 04:46:04,105 [__main__] [ERROR]  Need a UTF-8 locale. Currently 'iso8859-1'. Exiting...
2019-01-16 04:46:04,580 [__main__] [INFO ]  New locale set to: en_US.ISO8859-1
2019-01-16 04:46:04,580 [__main__] [ERROR]  Need a UTF-8 locale. Currently 'iso8859-1'. Exiting...
2019-01-16 04:48:16,864 [__main__] [INFO ]  New locale set to: en_US.UTF-8
2019-01-16 04:48:16,867 [__main__] [INFO ]  ###########################################
2019-01-16 04:48:16,867 [__main__] [INFO ]  #             *** pypo  ***               #
2019-01-16 04:48:16,867 [__main__] [INFO ]  #   Liquidsoap Scheduled Playout System   #
2019-01-16 04:48:16,867 [__main__] [INFO ]  ###########################################
```

So yeah I'm not sure why it doesn't get it set properly until a reboot.

Reply:
I see the issue - I need to include systemctl daemon reload at some point before the start of that. For some reason your xenial box is picking it up differently from mine... The override isn't being applied to yours...